workflows/cloud_run_job/main.tf

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+Create a Cloud Run job that processes data files in a Cloud Storage bucket.
+See https://cloud.google.com/workflows/docs/tutorials/execute-cloud-run-jobs
+before running the code snippet.
+*/
+
+# [START workflows_terraform_create_bucket]
+# Create a Cloud Storage bucket
+resource "google_storage_bucket" "default" {
+  name                        = "input-PROJECT_ID"
+  location                    = "us-central1"
+  storage_class               = "STANDARD"
+  force_destroy               = false
+  uniform_bucket_level_access = true
+}
+# [END workflows_terraform_create_bucket]
+
+# [START workflows_terraform_create_ar_repo]
+# Create an Artifact Registry repository
+resource "google_artifact_registry_repository" "default" {
+  location      = "us-central1"
+  repository_id = "REPOSITORY"
+  format        = "docker"
+}
+# [END workflows_terraform_create_ar_repo]
+
+# [START workflows_terraform_create_run_job]
+# Create a Cloud Run job
+resource "google_cloud_run_v2_job" "default" {
+  name     = "parallel-job"
+  location = "us-central1"
+
+  template {
+    task_count = 10
+    template {
+      containers {
+        image   = "us-central1-docker.pkg.dev/PROJECT_ID/REPOSITORY/parallel-job:latest"
+        command = ["python"]
+        args    = ["process.py"]
+        env {
+          name  = "INPUT_BUCKET"
+          value = "input-PROJECT_ID"
```

Comment:
Same goes for here. Replace project resource and replace `PROJECT_ID` like so https://github.com/terraform-google-modules/terraform-docs-samples/blob/3e9f924366befdeb26d1046a6479ed1e9c4d8bb6/run/pubsub/main.tf#L59

Reply:
Done